lib/cucumber/support_code/library.js

```diff
@@ -22,6 +22,15 @@ function Library(supportCodeDefinition) {
     });
   }
 
+  function getStackframe() {
+    var stackframes = StackTrace.getSync();
+    if (stackframes.length > 1) {
+      return stackframes[1];
```

Comment:
Based on the failing feature tests, I think this logic needs to actually be
```js
if (stackframes.length > 2) {
  return stackframes[2];
} // ...
```
Since the defineHook / defineStep is now calling, we have another function on the stack and thus need to look one level further back.